Cauli/CauliViewController/CauliViewController.swift

```diff
@@ -60,6 +60,11 @@ internal class CauliViewController: UITableViewController {
         let bundle = Bundle(for: SwitchTableViewCell.self)
         tableView.register(UINib(nibName: SwitchTableViewCell.nibName, bundle: bundle), forCellReuseIdentifier: SwitchTableViewCell.reuseIdentifier)
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
```

Comment:
This makes sure that the list of all florets is updated once the user goes back to this screen. The description of the `MapRemoteFloret` contains the number of enabled mappings. Without this change, the description will not be updated if the user enables or disables a mapping in the settings of the floret.